### PR TITLE
FH-4126 - Update Readme after restructuration

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-4126

### CODE REVIEW

**Comment by camilamacedo86** [comment]
Hi, @finp could you give a hand here in order to we move forward with this PR?

File: README.md [hunk 1]
```diff
@@ -17,27 +17,33 @@ Finally, install FHC bash completion: `fhc completion >> ~/.bashrc` (or ~/.zshrc
 ## Usage
 
 ### From the Command Line
-To see the list of commands available, just run `fhc`.   
+
+To see the list of commands available, just run `fhc`.
 See `fhc help` for general help, or `fhc help <someCommand>` for help on a specific command.
 
+#### Getting start
```
Comment (pb82) [inline README.md]: -> Getting started

File: README.md [hunk 2]
```diff
@@ -46,27 +52,19 @@ Then, you can require it in your code like so:
...
         console.log(projects);
       });
     });
 
-Some commands require params to be passed in - these are typically passed like so:
-
-    fhc.app.create({ title : 'Some title', project : 'someProjectId'}, function() {
-    });
-
+The commands require params to be passed in - these are typically passed like so:
```
Comment (pb82) [inline README.md]: Remove the `so` at the end

File: README.md [hunk 3]
```diff
@@ -46,27 +52,19 @@ Then, you can require it in your code like so:
...
 
-Older fhc commands still pass arguments in an ordered array, as below. The environment is still specified outside the array.
+    fhc.projects.list({ title : 'Some title', project : 'someProjectId'}, function() {});
 
-    fhc.app.logs({_ : ['projectId', 'appId'], env : 'dev' }, function() {
-    });
-
-*Observation* : The older commands have been changed to work as the new ones. In the future, all commands should work without the
-need to pass the params in an ordered array.
+*Note* : To force the result be as a json object use the parameter `json:true`.
```
Comment (pb82) [inline README.md]: -> To force the result to be a json object use the `json: true` parameter

**Comment by camilamacedo86** [comment]
@pb82 done the changes requested. Thank you for your help here. Could you check now for we are able to move forward with it?

**Comment by camilamacedo86** [comment]
Hi @pb82.  Please, could you check this one? All that as requested is done now. Could we move forward with it?

**Comment by camilamacedo86** [comment]
@davidffrench could you approve in order to allow me do: rebase with the master, bump version and merge?

**Comment by davidffrench** [comment]
@camilamacedo86 It is best to rebase locally and force push, once this branch is not out of date, i will approve the PR.

**Comment by camilamacedo86** [comment]
@davidffrench done 👍

**Pull request merged**